x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -1310,6 +1310,7 @@ export class EndpointDocGenerator {
         },
       ],
       status: 'installed',
+      release: 'ga',
```

Comment:
`release` key is [required by package-spec](https://github.com/elastic/package-spec/blob/master/versions/1/manifest.spec.yml#L247) and can be [one of `experimental`, `beta`, or `ga`](https://github.com/elastic/package-spec/blob/master/versions/1/manifest.spec.yml#L52-L58)